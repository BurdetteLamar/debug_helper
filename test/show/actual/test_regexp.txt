---
Regexp (message='test_regexp'):
  to_s: "(?-mix:(?<a>.)(?<b>.))"
  casefold?: false
  named_captures:
    a:
    - 1
    b:
    - 2
  encoding: !ruby/encoding US-ASCII
  fixed_encoding?: false
