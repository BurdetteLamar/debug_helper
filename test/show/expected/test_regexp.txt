---
Regexp (message='test_regexp'):
  Regexp#to_s: "(?-mix:(?<a>.)(?<b>.))"
  Regexp#casefold?: false
  Regexp#named_captures:
    a:
    - 1
    b:
    - 2
  Regexp#encoding: !ruby/encoding US-ASCII
  Regexp#fixed_encoding?: false
